### PR TITLE
fix ui in notification screen

### PR DESCRIPTION
**Pull request by IlyesBenAmara**
- this PR adds the UI corrections to the notifications screen

### CODE REVIEW

File: src/components/reviewMatches/NotificationWorklist.tsx [hunk 1]
```diff
@@ -55,11 +58,11 @@ const columns: GridColDef[] = [
     align: 'center',
     headerAlign: 'center',
     valueFormatter: (params: GridValueFormatterParams<number>) =>
-      params.value ? moment(params.value).format('YYYY-MM-DD') : null
+      params.value ? moment(params.value).format('DD/MM/YYYY') : null
```
Comment (MatthewErispe) [inline src/components/reviewMatches/NotificationWorklist.tsx:61]:
```suggestion
      params.value ? moment.unix(params.value).format('DD/MM/YYYY') : null
```

Comment (MatthewErispe) [reply src/components/reviewMatches/NotificationWorklist.tsx:61]: @IlyesBenAmara I think the value is a timestamp and needs the unix to convert ot date

**Review by MatthewErispe — COMMENTED** [review]
@IlyesBenAmara is it possilble to add some unit tests for the notifications worklist?

**Review by MatthewErispe — CHANGES_REQUESTED** [review]
@IlyesBenAmara are you able to add some unit tests for the notification worklist?

File: src/components/reviewMatches/NotificationWorklist.tsx [hunk 2]
```diff
@@ -55,11 +58,11 @@ const columns: GridColDef[] = [
     align: 'center',
     headerAlign: 'center',
     valueFormatter: (params: GridValueFormatterParams<number>) =>
-      params.value ? moment(params.value).format('YYYY-MM-DD') : null
+      params.value ? moment.unix(params.value).format('DD/MM/YYYY') : null
```
Comment (marrouchi) [inline src/components/reviewMatches/NotificationWorklist.tsx:61]: Would be better to transform any string to Date object at the fetch function level.
Then we could add a helper function to format any date, so we keen consistent format across the screens.

**Review by marrouchi — CHANGES_REQUESTED** [review]

File: src/utils/formatters.ts [hunk 3]
```diff
@@ -4,7 +4,7 @@ import { AnyRecord, ValueOf } from '../types/PatientRecord'
 
 export const formatDate = (value: unknown) => {
   if (typeof value === 'object') {
-    return moment(value).format('YYYY-MM-DD')
+    return moment(value).format('DD/MM/YYYY')
```
Comment (marrouchi) [inline src/utils/formatters.ts:7]: Could we add a else condition ?

File: src/utils/formatters.ts [hunk 4]
```diff
@@ -4,7 +4,7 @@ import { AnyRecord, ValueOf } from '../types/PatientRecord'
 
 export const formatDate = (value: unknown) => {
```
Comment (marrouchi) [inline src/utils/formatters.ts:5]: Let's define the type here "Date"

File: src/types/Notification.ts [hunk 5]
```diff
@@ -2,10 +2,10 @@ export default interface Notification {
   id: string
   type: string
   reason?: string | null
-  created: number
+  created: string | Date
```
Comment (marrouchi) [inline src/types/Notification.ts:5]:
```suggestion
  created: Date
```
Let's be strict about the type.

**Review by marrouchi — CHANGES_REQUESTED** [review]

**Review by MatthewErispe — APPROVED** [review]

**Pull request merged**